### PR TITLE
Add logstash as output to the same config file of each beat

### PR DESCRIPTION
**Pull request by monicasarbu**

### CODE REVIEW

File: etc/libbeat.yml [hunk 1]
```diff
@@ -98,27 +98,45 @@ output:
...
+
+    # Optional load balance the events between the Logstash hosts
+    # loadbalance: true
+
+    # Optional index name. The default index name is logstash
+    # index: logstash
+
+    # Optional TLS. The default is on
+    # tls:
+    #   disabled: true
```
Comment (andrewkroh) [inline etc/libbeat.yml]: I think the other TLS options from ES should also be included here.

File: etc/libbeat.yml [hunk 2]
```diff
@@ -121,6 +121,26 @@ output:
     #reconnect_interval: 1
 
 
+  ### Logstash as output
+
```
Comment (andrewkroh) [inline etc/libbeat.yml]: Having this newline is inconsistent with the formatting of the others (elasticsearch, redis, file).

File: etc/libbeat.yml [hunk 3]
```diff
@@ -121,6 +121,26 @@ output:
...
+    # Optional load balance the events between the Logstash hosts
+    #loadbalance: true
+
+    # Optional index name. The default index name is logstash
+    #index: logstash
+
+    # Optional TLS. The default is on
+    #tls:
+      #disabled: true
+
```
Comment (andrewkroh) [inline etc/libbeat.yml]: I think the other TLS options from ES should also be included here.

File: etc/libbeat.yml [hunk 4]
```diff
@@ -121,6 +121,26 @@ output:
...
+    # The Logstash hosts
+    #hosts: ["localhost:5044", "logstash:5045"]
+
+    # Optional load balance the events between the Logstash hosts
+    #loadbalance: true
+
+    # Optional index name. The default index name is logstash
+    #index: logstash
+
+    # Optional TLS. The default is on
```
Comment (tsg) [inline etc/libbeat.yml]: I think I know what you are referring to, but the "the default is on" is missleading. If you leave everything disabled, tls is off, right?

**Comment by andrewkroh** [comment]
Should redis be completely removed per #160?

File: etc/libbeat.yml [hunk 5]
```diff
@@ -121,6 +121,26 @@ output:
...
+    #enabled: true
+
+    # The Logstash hosts
+    #hosts: ["localhost:5044", "logstash:5045"]
+
+    # Optional load balance the events between the Logstash hosts
+    #loadbalance: true
+
+    # Optional index name. The default index name is logstash
+    #index: logstash
```
Comment (tsg) [inline etc/libbeat.yml]: The default is the beat name, not "logstash", see: https://www.elastic.co/guide/en/beats/libbeat/master/configuration.html#_index_2

File: etc/libbeat.yml (hunk 4 above)
Comment (monicasarbu) [inline etc/libbeat.yml]: the disabled: true option is tricky. It tricked me as well even if I knew it before.

**Comment by monicasarbu** [comment]
Ready to merge.

**Pull request merged**